openedx/core/djangoapps/discussions/models.py

```diff
@@ -159,6 +174,23 @@ class Features(Enum):
             '',
             '',
         )._asdict(),
+        "messages": ProviderMessages(
+            setup=[
+                _(
+                    '{provider} requires that LTI advanced sharing be enabled for your course'
```

Comment:
there is no space after course and before comma. 
```
... your course,
 as this provider uses email address and username to personalize
```

Reply:
This message should be reused for `Yellowdig` as per description in ticket.

Reply:
This message is also needed for `InScribe`. If you make this reusable message, then both `Discourse` and `InScribe` can use it. 

Here is the ticket for that:
https://openedx.atlassian.net/browse/TNL-8523